test/IDE/complete_value_expr.swift

```diff
@@ -2185,3 +2186,16 @@ func testWrapSuccess(promise: Int, seal: Resolver<Void>) {
   // COMPLETE_CALL_RESULT: Pattern/CurrModule:                 ({#Void#}, {#Bool#})[#Void#]; name=(Void, Bool)
   // COMPLETE_CALL_RESULT: End completions
 }
+
+protocol BrokenConformanceP {
+  static func staticFunc()
+  func instanceFunc()
+}
+struct BrokenConformanceS: BrokenConformanceP {
+}
+func brokenConformance(arg: BrokenConformanceS) {
+  arg.#^BROKEN_CONFORMANCE^#
```

Comment:
I am absolutely sure I saw a similar test with a broken conformance somewhere, with a descriptive comment. I was going to extend that test to guard against static requirements, but have had no luck searching so far..